tools/testsys/Test.toml.example

```diff
@@ -4,15 +4,38 @@
 # This section contains configuration details for all testing
 [test]
 
+# Configurable values:
```

Comment:
These shouldn't be in the `[test]` table since they cannot be added there. I still prefer them at the bottom.